_posts/2021-02-26-feather-nrf52840.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: blog
+title: How to Port a Board for RIOT
+authors:
+- name: Martine S. Lenders
+  email: mlenders@riot-os.org
+  affiliation: Freie Universität Berlin
+  githubid: miri64
+  twitterid: miri_64
+  stackoverflowid: 11921757
```

Comment:
There is a bug that we did not spot in #21 for this link. See https://github.com/RIOT-OS/riot-os.org/pull/21#pullrequestreview-600545886. Should I just fix it here, or do we do this in a separate PR?

Reply:
Feel free to piggyback that here

Reply:
Fix provided in #32